service/src/main/kotlin/fi/espoo/evaka/placement/PlacementQueries.kt

```diff
@@ -373,6 +373,77 @@ fun Handle.getDaycareGroupPlacements(
         .toList()
 }
 
+fun Handle.getMissingGroupPlacements(
+    unitId: UUID
+): List<MissingGroupPlacement> {
+    // language=SQL
+    val sql =
+        """
+        WITH gaps AS (
+            SELECT placement.id AS placement_id, gap * daterange(placement.start_date, placement.end_date, '[]') AS gap
```

Comment:
Reminder: when this is merged, check from production whether there is performance issues here